client/my-sites/menus/item-options/options.jsx

```diff
@@ -30,10 +31,18 @@ var Options = React.createClass( {
 
 	renderItem: function( item ) {
 		var checked = ( item.ID === this.props.item.content_id ) &&
-			( this.props.itemType.name === this.props.item.type ),
-			isPrivate = item.status && item.status === 'private';
+			( this.props.itemType.name === this.props.item.type );
+
+		const classes = [];
+		if ( item.status && item.status === 'private' ) {
+			classes.push( 'option-is-private' );
+		}
+		if ( isInjectedNewPageItem( item ) ) {
+			classes.push( 'create-new-item' );
+		}
+
 		return (
-			<li key={ this.props.itemType.name + '-' + item.ID } className={ isPrivate ? 'option-is-private' : '' } >
+			<li key={ this.props.itemType.name + '-' + item.ID } className={ classes.join( ' ' ) } >
```

Comment:
You may find that using [classnames](https://www.npmjs.com/package/classnames) is a bit clearer

```javascript
const classes = classNames( {
	'option-is-private': item.status === 'private',
	'create-new-item': isInjectedNewPageItem( item )
} );
<li className={ classes }>/...</li>
```